.travis.yml

```diff
@@ -8,5 +8,7 @@ env:
   - ROS_DISTRO="noetic"
 install:
   - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config
+  - pip install flatten-dict --user
```

Comment:
We should use `rosdep` to resolve dependencies otherwise we cannot release the library

Reply:
Since `flatten-dict` is not released as a `debian` package, we cannot depend on this package and it should be included in the source of `rosbag_pandas`

Reply:
OK, so which way you would prefer? `git submodule` or include the most current version in the source tree?

Reply:
Including this version: https://github.com/ianlini/flatten-dict/releases/tag/0.3.0 in the source tree would be fine